.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Can you remove this file? Or at least add the `.vscode` in `.gitignore`, my vim god tells me that it does not belong here 😄

Reply:
removed